auth/auth.go

```diff
@@ -5,13 +5,32 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/url"
-	"strings"
 
 	"github.com/G-Node/gin-cli/util"
 	"github.com/G-Node/gin-cli/web"
 	"github.com/G-Node/gin-core/gin"
+	"net/http"
```

Comment:
Right, in that case yeah, using it for codes is much nicer than checking against a literal.